app/javascript/Pages/Classes.jsx

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from "react";
+import Layout from "./Layout";
+
+class Classes extends Component {
+  render() {
+    return (
+      <Layout current_user={this.props.current_user}>
```

Comment:
**suggestion (non-blocking)**: It's a good practice to declare the props a component expects, as that allows anyone who sees the component to know how to use it. Take a look at https://reactjs.org/docs/typechecking-with-proptypes.html